src/Explorer/Explorer.tsx

```diff
@@ -1728,60 +1581,6 @@ export default class Explorer {
     this._addSynapseLinkDialogProps.valueHasMutated();
   };
 
-  public handleMessage(message: any) {
```

Comment:
Do we no longer need this? I remember this function handles opening the gettings started notebook when user clicks on the "setup container" button in the quickstart blade. Is that being handled somewhere else now or is the worflow changed/removed?

Reply:
Good question. I will try it out. It may be broken.

Reply:
I was able to fix open actions